src/session.rs

```diff
@@ -49,18 +49,46 @@ pub struct Cookie {
     secure: Option<bool>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct Header {
+    key: String,
```

Comment:
HTTPie calls these `name`, not `key`, so they fail to load each other's sessions. Maybe we can copy a file from HTTPie's test suite?

Reply:
Yup, that is a good idea 👍

Reply:
**Update:** I haven't used a verbatim test file from HTTPie since we are not yet 100% compatible with the current session format. I am planning to address this in a follow-up PR that adds support for the new style cookies introduced in https://github.com/httpie/httpie/pull/1312

Reply:
> not yet 100% compatible

Do you mean that we crash or that we interpret the session files incorrectly?

Reply:
Actually, there are test files that mix the new header format with the old cookie format. See https://github.com/httpie/httpie/blob/master/tests/fixtures/session_data/old/cookies_dict_with_extras.json